src/zenml/zen_stores/migrations/versions/6119cd9b93c2_tags_table.py

```diff
@@ -107,7 +106,7 @@ def upgrade() -> None:
         for model_id_, tags_in_model in model_tags_prepared:
             for tag in tags_in_model:
                 insert_tag_models += (
-                    f"('{_get_tag_resource_id(tags_ids_mapping[tag],model_id_).hex}', "
+                    f"('{uuid4().hex}', "
```

Comment:
Updating old migration just to make sure users upgrading from the old versions will not experience issues here.